rest/utilities_testing.go

```diff
@@ -1198,6 +1211,7 @@ func Request(method, resource, body string) *http.Request {
 	if err != nil {
 		panic(fmt.Sprintf("http.NewRequest failed: %v", err))
 	}
+	request.RemoteAddr = ":0"     // usually populated by actual HTTP requests going into a HTTP Server
```

Comment:
Take or leave:

Does this make sense to use a fake number that couldn't be a real port number? I think it would have to be an int, maybe 65536? We could use a string if the int isn't parsed.

Reply:
I'll put `test.client.addr:99999` to make it even more obvious it's a fake RemoteAddr value. `SplitHostPort` doesn't do any validation of port numbers so it's safe.